src/themes/default/data/index.js

```diff
@@ -35,16 +35,30 @@ export default ({
           items: [
             hasQueries
               ? {
-                  name: 'Queries',
```

Comment:
Thes changes are just to highlight the issue. They will be reverted in the theme before being merged.